docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_the_cli.rst

```diff
@@ -3,26 +3,172 @@
+
+        or in the non-interactive mode:
+
+        .. code-block:: bash
+
+            great_expectations --v3-api suite new --no-jupyter
+
+
+        3. One of the easiest ways to get starting in the interactive mode is to take advantage of the `--profile` flag:
+
+        .. code-block:: bash
+
+            great_expectations --v3-api suite new --interactive --profile
+
+        This instructs Great Expectations to inspect your Batch of data and suggest the initial set of Expectations.
+        When the notebook opens (or when you open it later if the above command is used with the `--no-jupyter` flag),
+        you can edit these expectations.  For example, you might tighten some tolerances, compared to the initial values,
+        based on your knowledge of your dataset.  After that, you can review and save your Expectation Suite and run validations using it.
+
+        Here are some general guidelines for taking advantage of the profiling capability for bootstrapping your Expectation Suite.
+
+        Within the notebook, run the first cell in the notebook that loads the data.  (This step was explained above.)
+
+        The next code cell in the notebook presents you with a list of all the columns found in your selected data:
+
+        .. code-block:: python
+
+            ignored_columns = [
+                "Name",
+                "Age",
+                "Address",
+                "Occupation",
+                ...
+            ]
+
+        By default, all columns are ignored. To select which columns you want to be profiled for generating Expectations on, simply comment them out to include them.
+
+        The next code cell is where you will configure and instantiate your profiler, and build your suite. You can leave these defaults as-is for now - :ref:`learn more about the available parameters here. <how_to_guides__creating_and_editing_expectations__how_to_create_an_expectation_suite_with_the_user_configurable_profiler>`
+
+        When you run this cell and build your suite, you will see a list of the expectations included by column. At this point, you may also make modifications to the ignored_columns or the profiler, and re-run the cell.
+
+        .. code-block:: python
+
+            profiler = UserConfigurableProfiler(
+                profile_dataset=validator,
+                excluded_expectations=None,
+                ignored_columns=ignored_columns,
+                not_null_only=False,
+                primary_or_compound_key=False,
+                semantic_types_dict=None,
+                table_expectations_only=False,
+                value_set_threshold="MANY",
+            )
+            suite = profiler.build_suite()
+
+        Finally, run the next few code cells to see the automatically generated Expectation Suite in Data Docs.
+
+        Because the profiler-based expectations are too permissive (i.e., lax tolerances), you will want to edit this
```

Comment:
```suggestion
        Because the profiler-based expectations have strict tolerances, you will want to :ref:`edit<how_to_guides__creating_and_editing_expectations__how_to_create_an_expectation_suite_with_the_user_configurable_profiler>` this
```